site/docs/v1/tech/tutorials/integrate-laravel-api.md

```diff
@@ -0,0 +1,233 @@
+
+```php
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-laravel-api/main/laravel/app/FusionAuth/Providers/FusionAuthEloquentUserProvider.php %}
+```
+
+Now, add an [Authentication Guard](https://laravel.com/docs/10.x/authentication#adding-custom-guards) in `app/FusionAuth/FusionAuthJWTGuard.php` to call that custom method from the User Provider created above.
+
+```php
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-laravel-api/main/laravel/app/FusionAuth/FusionAuthJWTGuard.php %}
+```
+
+To override the existing classes when the ones you just added, you must create a [Service Provider](https://laravel.com/docs/10.x/providers) in `app/FusionAuth/Providers/FusionAuthServiceProvider.php`.
+
+```php
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-laravel-api/main/laravel/app/FusionAuth/Providers/FusionAuthServiceProvider.php %}
+```
+
+### Retrieving Public Key
+
+To make your API trust JWTs issued by FusionAuth, you must import the Public Key from your FusionAuth application into it.
+
+If you have [jq](https://stedolan.github.io/jq/download/) _(a script to parse JSON objects)_ installed, you can run the command below to fetch and save it to `storage/public-key.pem`.
+
+```shell
+curl -H 'Authorization: this_really_should_be_a_long_random_alphanumeric_value_but_this_still_works' http://localhost:9011/api/key/1afa4d7e-76f0-45e9-bb46-98be5329ef37 | jq -r '.key.publicKey' > storage/public-key.pem
+```
+
+If you don't have it, log into the [FusionAuth admin screen](http://localhost:9011) using the admin user credentials ("admin@example.com"/"password"), navigate to <span>Settings -> Key Master</span>{:.breadcrumb}, locate the key named `For exampleapp` and click its download button. Inside the downloaded `.zip` file, go to the `keys` folder and extract `public-key.pem` to the `storage` folder in your Laravel app.
+
+### Creating a Controller
+
+Create a controller in `app/Http/Controllers/Api/MessagesController.php` which gives back a JSON message.
+
+```php
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-laravel-api/main/laravel/app/Http/Controllers/Api/MessagesController.php %}
+```
+
+### Testing the Authentication Flow
+
+Finally, start your application.
+
+```shell
+./vendor/bin/sail up -d
+```
+
+Make sure that everything is running fine by making a request to [localhost/api/auth/me](http://localhost/api/auth/me).
+
+```shell
+curl -X POST -H 'Accept: application/json' http://localhost/api/auth/me
+```
+
+You should see an error message because you're not providing any JWT to the request.
+
+```json
+{"message":"Unauthenticated."}
+```
+
+Now, you need to call FusionAuth to get an access token. For ease of use, these instructions will use the Login API, but you could also get the access token via the hosted login pages.
```

Comment:
Do you need to modify this to not use the Login API now?

Reply:
Yes, I forgot to change this. Done. Thanks!